stories/scss/navSide/navSide.stories.html

```diff
@@ -0,0 +1,92 @@
+<div class="navSide" [class.mod-withBanner]="withBanner" [class.mod-compact]="compact" [class.is-open]="mobileMenuOpened">
+	<nav role="navigation" [attr.aria-label]="titleNav" class="navSide-wrapper">
+		<div class="navSide-mainSection">
+			<div class="navSide-item mod-mobileToggle" *ngIf="mobileToggle">
+				<button type="button" [attr.aria-expanded]="mobileMenuOpened" class="navSide-item-link" (click)="$event.preventDefault(); mobileMenuOpened=!mobileMenuOpened">
+					<span class="navSide-item-link-title">Menu</span>
+					<span aria-hidden="true" class="navSide-item-arrow lucca-icon icon-arrowEast"></span>
+				</button>
+			</div>
+
+			<ng-container *ngIf="loading; else menu">
+				<div aria-hidden="true">
+					<div class="navSide-item-placeholder" [class.mod-inverted]="loadingInverted"></div>
+					<div class="navSide-item-placeholder" [class.mod-inverted]="loadingInverted"></div>
+					<div class="navSide-item-placeholder" [class.mod-inverted]="loadingInverted"></div>
+				</div>
+			</ng-container>
+
+			<ng-template #menu>
+				<ul class="navSide-scrollWrapper">
+					<li class="navSide-item" [class.is-open]="firstSubMenuOpened">
+						<button type="button" class="navSide-item-link" [attr.aria-expanded]="firstSubMenuOpened" (click)="$event.preventDefault(); firstSubMenuOpened=!firstSubMenuOpened">
+							<span aria-hidden="true" class="lucca-icon icon-heart"></span>
+							<span class="navSide-item-link-title">Section 1</span>
+							<span aria-hidden="true" class="navSide-item-arrow lucca-icon icon-arrowEast"></span>
+						</button>
+						<ul class="navSide-item-subMenu">
+							<li class="navSide-item-subMenu-item" [class.u-animatedSlideInLeft]="firstSubMenuOpened">
+								<a href="#" class="navSide-item-subMenu-link" (click)="$event.preventDefault()">Subsection #1.1</a>
```

Comment:
Should'nt those be named `navSide-item-subMenu-item-link` ?

Reply:
Yes, but it would introduce a regression…